packages/docs/site/docs/developers/24-limitations/01-index.md

```diff
@@ -16,7 +16,19 @@ Playground [disables network connections](/blueprints/data-format#features) by d
 
 ### Temporary by design
 
-As Playground [streams rather than serves](/about#streamed-not-served) WordPress, all database changes and uploads will be gone when you refresh the page. To avoid losing your work, either [export your work](/quick-start-guide#save-your-site) before or enable storage in the browser/device via the "Save" button in the UI.
+As Playground [streams rather than serves](/about#streamed-not-served) WordPress, all database changes and uploads will be gone when you refresh the page. To avoid losing your work, either [export your work](/quick-start-guide#save-your-site) before or enable storage in the browser/device via the "Save" button found in the "Open Site Manager" menu on the top left of the site.
+
+<details>  
+  <summary>Click for a screenshot</summary>    
+  <div>    
+	 <img width="406" alt="Open Site Manager" src="https://github.com/user-attachments/assets/f66e4116-069c-4764-9305-3cec03eb72d5" />
+  </div>
+	<br>
+	<div>
+<img width="408" alt="Save Button" src="https://github.com/user-attachments/assets/a677fbff-2346-4579-9543-56e33e8e42b9" />
+
+  </div>    
+</details>
```

Comment:
We use Docusaurus to build our documentation and unfortunately, this approach [breaks the build](https://github.com/WordPress/wordpress-playground/actions/runs/13273463554/job/37058148867?pr=2145).

[Here is an example](https://github.com/jswhisperer/wordpress-playground/blob/patch-1/packages/docs/site/docs/developers/05-local-development/02-vscode-extension.md?plain=1#L28-L31) of how you can add images to the documentation. 

Why did you decide to collapse the image? 
It would be easier to discover the save feature if the image was always visible.

Reply:
Aw I'm familiar with docusaurus, sorry my markdown is rusty will adjust.
Happy to leave it expanded I just didn't want to disrupt the current docs :P